lib/late_ack_tracker.c

```diff
@@ -198,6 +198,9 @@ void
 late_ack_tracker_free(AckTracker *s)
 {
   LateAckTracker *self = (LateAckTracker *)s;
+
+  g_static_mutex_free(&self->storage_mutex);
+
   guint32 count = ring_buffer_count(&self->ack_record_storage);
```

Comment:
Pls move this past the declaration in the next line.

Probably ring buffer count wouldnt use the just freed mutex, but I prefer strong visual separation of declarations vs code.